packages/next-app/src/app/daoadmin/action/[actionId]/(components)/ActionAttributeContent.tsx

```diff
@@ -84,24 +115,20 @@ export function ActionAttributeContent() {
               <div className="flex flex-col gap-4">
                 {selectedAction?.fields.map((field) => {
                   return (
-                    <div key={field.name}>
+                    <div key={field.key}>
                       <Input
-                        label={field.name}
+                        label={field.label}
                         placeholder={field.placeholder}
-                        {...register(toCamelCase(`${field.name}`))}
+                        {...register(field.key, {
+                          validate: field?.getValidations?.(chain),
+                        })}
+                        errorMessage={
+                          _.isEmpty(formState.errors[field.key])
+                            ? undefined
+                            : String(formState.errors[field.key]?.message)
+                        }
```

Comment:
if you wanted to do optional chaining you could also write the following
```suggestion
                        errorMessage={formState.errors?.[field.key]?.message}
```